pattern-1/conf/is/conf/user-mgt.xml

```diff
@@ -41,7 +41,7 @@
         <UserStoreManager class="org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager">
             <Property name="TenantManager">org.wso2.carbon.user.core.tenant.JDBCTenantManager</Property>
             <Property name="driverName">com.mysql.jdbc.Driver</Property>
-            <Property name="url">jdbc:mysql://wso2is-pattern1-mysql-service:3306/WSO2_USER_STORE_DB?autoReconnect=true&amp;verifyServerCertificate=false&amp;useSSL=true</Property>
+            <Property name="url">jdbc:mysql://wso2is-pattern1-mysql-service:3306/WSO2_USER_DB?autoReconnect=true&amp;verifyServerCertificate=false&amp;useSSL=true</Property>
```

Comment:
we will have to make useSSL to false, otheriwse newer connectors will fail in an out-of-the-box setup ?